src/Qir/Runtime/public/CoreTypes.hpp

```diff
@@ -33,7 +33,7 @@
 // is a unique type in the QIR.
```

Comment:
In the comment above "Qubit" type is referenced. We need to change this comment.